ga4gh/cli.py

```diff
@@ -66,6 +66,7 @@ def server_main():
             args.dataDir, args.variantSetClass)
         frontend.app.run(host="0.0.0.0", port=args.port, debug=True)
 
+
```

Comment:
why this change?  generally, let's try to keep the diff as small as possible, and especially not add unnecessary files to the diff (unless this is some pep8 thing?)